int-tests/src/test-integration/java/io/neow3j/compiler/StorageIntegrationTest.java

```diff
@@ -1238,7 +1238,7 @@ public static io.neow3j.devpack.List<ByteString> findWithFindOptionValuesOnly()
 
         public static io.neow3j.devpack.Map<ByteString, ByteString> findWithFindOptionDeserializeValues() {
             Storage.put(ctx, hexToBytes("0102"), hexToBytes("102030"));
-            byte findOption = FindOptions.DeserializeValues & FindOptions.PickField0;
+            int findOption = FindOptions.DeserializeValues & FindOptions.PickField0;
```

Comment:
Don't you think we should have at least one test that tests the new Backwards FindOption?
After all the discussions we had, I don't trust this stuff.

Reply:
Yes, I thought about adding an integration test. Though, I’ll need to wait for neo3-privatenet-docker to update to Neo v3.6.0 including this new FindOption.

Adding a unit test doesn‘t add any value imo.

If you agree, please change this PR to a draft PR. Then, I‘ll await the Neo 3.6.0 release to continue on this issue.

Reply:
ah ye right. I always forget how tedious our dependencies for a proper release are.

Reply:
It seems that I can't change the PR to a draft. @mialbu can you do that?